backend/app/routers/authentication/authentication.py

```
@@ -0,0 +1,204 @@
+    return {"message": "Email or phone verified successfully"}
+
+
+@router.get(
+    "/protected-endpoint",
+    response_model=dict,
+    responses={
+        401: {"description": "Not authenticated"},
+        403: {"description": "Not authorized"},
+    },
+)
+def protected_route(current_user: User = Depends(get_current_user)) -> dict:
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)